build/Packaging/NugetAgent/NewRelic.Agent.nuspec

```diff
@@ -26,6 +26,8 @@
     </contentFiles>
   </metadata>
   <files>
+    <file src="**/newrelic/**/*" target="." />
```

Comment:
Nit: Might as well make the slash  / backslash usage consistent everywhere...

Reply:
Good catch, done.